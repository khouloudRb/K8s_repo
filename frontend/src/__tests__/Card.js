import React from 'react';
import CardBack from '../components/MoviesTable/MovieCard/CardBack';
import { shallow, mount } from 'enzyme';

describe('<CardBack>', () => {
   it('renders ', () => {
   const wrapper = shallow(<CardBack description="desciption pour un test"/>);
   expect (wrapper.exists()).toBe(true);
   expect (wrapper).toMatchSnapshot();

   
   
   });

  // ...
});