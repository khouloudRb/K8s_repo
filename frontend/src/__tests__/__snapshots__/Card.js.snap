// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardBack> renders  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardBack
    description="desciption pour un test"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h5>
           Summary 
        </h5>,
        <p>
           
          desciption pour un test
           
        </p>,
      ],
      "className": "back",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Summary ",
        },
        "ref": null,
        "rendered": " Summary ",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            " ",
            "desciption pour un test",
            " ",
          ],
        },
        "ref": null,
        "rendered": Array [
          " ",
          "desciption pour un test",
          " ",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h5>
             Summary 
          </h5>,
          <p>
             
            desciption pour un test
             
          </p>,
        ],
        "className": "back",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Summary ",
          },
          "ref": null,
          "rendered": " Summary ",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              "desciption pour un test",
              " ",
            ],
          },
          "ref": null,
          "rendered": Array [
            " ",
            "desciption pour un test",
            " ",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
